docs/api-reference.md

```diff
@@ -226,6 +289,48 @@ class App {
 
 ```
 
+
+`getAuth(): FirebaseAuthState` : Returns the client's current Authentication State.
```

Comment:
FirebaseAuthState is an interface in the definition file auth_backend.d.ts. I added the method signature in documentation, which might be helpful, incase someone is digging for more information. 
I think it is similar to https://firebase.google.com/docs/reference/js/firebase.auth.AuthCredential , but I am not sure, hence didn't link it. Please verify and let me know as well. Thanks.,